pkgs/by-name/cl/cling/package.nix

```diff
@@ -34,41 +34,41 @@
 let
   stdenv = clangStdenv;
 
-  # The patched clang lives in the LLVM megarepo
-  clangSrc = fetchFromGitHub {
+  version = "1.2";
+
+  clingSrc = fetchFromGitHub {
     owner = "root-project";
-    repo = "llvm-project";
-    # cling-llvm13 branch
-    rev = "3610201fbe0352a63efb5cb45f4ea4987702c735";
-    sha256 = "sha256-Cb7BvV7yobG+mkaYe7zD2KcnPvm8/vmVATNWssklXyk=";
-    sparseCheckout = [ "clang" ];
+    repo = "cling";
+    rev = "v${version}";
+    sha256 = "sha256-ay9FXANJmB/+AdnCR4WOKHuPm6P88wLqoOgiKJwJ8JM=";
   };
 
-  llvm = llvmPackages_13.llvm.override { enableSharedLibraries = false; };
+  llvm = llvmPackages_18.llvm.override { enableSharedLibraries = false; };
```

Comment:
FWIW, it didn’t seem like disabling `enableSharedLibraries` was necessary; you could probably save an LLVM build by skipping this. (Actually I’m a bit puzzled by the `-resource-dir` flag in general, since the directory doesn’t contain the usual Clang resource directory stuff, but… maybe I just don’t understand Cling.)

(Not a blocker, as not a new issue.)

Reply:
This was a fix for #306782 but `xeus-cling` seems to work without it now. I'll remove it.